core/src/trezor/workflow.py

```diff
@@ -215,7 +237,7 @@ def set(self, timeout_ms: int, callback: IdleCallback) -> None:
 
```

Comment:
Again, the docstring above `idle_timer.set() also counts as an activity, so all running idle timers are reset.` needs updating if I understand it correctly.

Reply:
Also the comment below the docstring `The reason for counting set() [...]` is probably invalidated by these changes.